docs/source/tutorial/uncertainty.rst

```diff
@@ -0,0 +1,7 @@
+Analysis of Uncertainty
+=======================
+.. todo::
+
+    @mberr please give a bit of narrative on why you would want to use the uncertainty predictions,
```

Comment:
@mberr narrative goes here, please and thank you!

Reply:
alternative, make a new module in the same place as `predict.py` and put this all as the module-level docstring